tdp/cli/commands/init.py

```diff
@@ -31,19 +36,49 @@
     multiple=True,
     help="Path to TDP variables overrides. Can be used multiple times. Last one takes precedence.",
 )
+@click.argument(
+    "alembic_ini_path",
+    envvar="ALEMBIC_CONFIG",
+    required=True,
+    type=click.Path(exists=True, resolve_path=True, path_type=Path),
+)
 @collections_option
-@database_dsn_option
+@database_dsn_option(create_engine=False)
 @validate_option
 @vars_option(exists=False)
 def init(
     overrides: tuple[Path],
     collections: Collections,
-    db_engine: Engine,
+    alembic_ini_path: Path,
+    db_engine: str,
```

Comment:
This parameter should be `database_dsn` if the `create_engine=False` is passed to the `@database_dsn_option`.